examples/src/components/GeoPushpinChartConfigurationExample.jsx

```diff
@@ -0,0 +1,77 @@
+// (C) 2020 GoodData Corporation
+import React, { Component } from "react";
+import { GeoPushpinChart, Model } from "@gooddata/react-components";
+
+import "@gooddata/react-components/styles/css/main.css";
+
+import { projectId } from "../utils/fixtures";
+import {
+    MAPBOX_TOKEN,
+    cityCoordinatesUri,
+    populationUri,
+    densityUri,
+    stateNamesUri,
+    cityNamesUri,
+} from "../utils/fixturesGeoChart";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const colorMeasure = Model.measure(densityUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Density");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const segmentByAttribute = Model.attribute(stateNamesUri).localIdentifier("segment");
+const toolTipTexAttribute = Model.attribute(cityNamesUri).localIdentifier("tooltipText");
```

Comment:
`toolTipTexAttribute` => `tooltipTextAttribute`

Reply:
fixed

Reply:
@dlcd2203 please use the lowercase for letter `t` in `toolTipTextAttribute`
`toolTipTextAttribute` => `tooltipTextAttribute `